pkg/injector/bootstrap.go

```diff
@@ -3,21 +3,30 @@ package injector
 import (
 	"context"
 	"fmt"
+	"strings"
+	"time"
 
 	xds_bootstrap "github.com/envoyproxy/go-control-plane/envoy/config/bootstrap/v3"
 	"github.com/google/uuid"
 	corev1 "k8s.io/api/core/v1"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/client-go/util/retry"
 
 	"github.com/openservicemesh/osm/pkg/certificate"
 	"github.com/openservicemesh/osm/pkg/constants"
 	"github.com/openservicemesh/osm/pkg/envoy/bootstrap"
 	"github.com/openservicemesh/osm/pkg/errcode"
+	"github.com/openservicemesh/osm/pkg/k8s"
 	"github.com/openservicemesh/osm/pkg/models"
 	"github.com/openservicemesh/osm/pkg/utils"
 	"github.com/openservicemesh/osm/pkg/version"
 )
 
+const (
+	signingIssuerID    = "signing_issuer_id"
+	validatingIssuerID = "validating_issuer_id"
```

Comment:
What's the purpose of these constants? Are they keys for the secret? If so, that should be clear based on the var name